zephyr/include/rtos/interrupt.h

```diff
@@ -49,8 +49,8 @@ static inline void interrupt_unregister(uint32_t irq, const void *arg)
 static inline int interrupt_get_irq(unsigned int irq, const char *cascade)
 {
 #if defined(CONFIG_LIBRARY) || defined(CONFIG_ACE) || defined(CONFIG_CAVS) || \
-	defined(CONFIG_ZEPHYR_POSIX) || (defined(CONFIG_IMX) && !defined(CONFIG_IMX8M)) \
-	|| defined(CONFIG_AMD)
+	defined(CONFIG_ZEPHYR_POSIX) || \
+	(defined(CONFIG_IMX) && !defined(CONFIG_IMX8M)) || defined(CONFIG_AMD)
```

Comment:
please, make commit titles and preferably bodies a bit more descriptive, at least mention AMD and Zephyr